### PR TITLE
#243 done

### PR DESCRIPTION
**Pull request by MSKose**

### CODE REVIEW

File: apps/api/src/api/art/content-types/art/lifecycles.ts [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+
+    // populating artist to use in email subject
+    const populatedArtist = await strapi.entityService.findOne('api::art.art', result.id, {
+      populate: ['artist'],
+    },)
+    const artist = populatedArtist.artist
+
+    if (editorEmails?.length > 0) {
+      strapi.plugins['email'].services.email.send({
+        to: editorEmails,
```
Comment (7alip) [inline apps/api/src/api/art/content-types/art/lifecycles.ts:14]: Does `to: editorEmails` work with array format? Maybe it expects a string value separated by comma, I'm not sure.

File: apps/api/src/api/art/content-types/art/lifecycles.ts [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
-      from: 'info@wsvvrijheid.nl',
-      subject: `New Art Created ${result.title}`,
-      html: `<p><a href="https://api.wsvvrijheid.nl/admin/content-manager/collectionType/api::art.art${result.id}">${result.title}</a></p>`,
-    })
+  async afterCreate({ result }) {
+    // do not forget to define your comma seperated EDITOR_EMAILS in your local env
+    const editorEmails = process.env.EDITOR_EMAILS?.split(',');
+
+    // populating artist to use in email subject
+    const populatedArtist = await strapi.entityService.findOne('api::art.art', result.id, {
```
Comment (7alip) [inline apps/api/src/api/art/content-types/art/lifecycles.ts:7]: `populatedArt` could be better since you retrieve it from arts data.

**Review by 7alip — COMMENTED** [review]
Thanks

File: apps/api/src/api/art/content-types/art/lifecycles.ts (hunk 1 above)
Comment (MSKose) [reply apps/api/src/api/art/content-types/art/lifecycles.ts:14]: I was unsure too as the [docs](https://docs.strapi.io/developer-docs/latest/plugins/email.html#sending-emails-with-a-controller-or-service) stated that it takes `string` but it is working as an `array` indeed.

Comment (7alip) [reply apps/api/src/api/art/content-types/art/lifecycles.ts:14]: So, if you tested it and it worked, you can merge it.

File: apps/api/src/api/art/content-types/art/lifecycles.ts (hunk 2 above)
Comment (MSKose) [reply apps/api/src/api/art/content-types/art/lifecycles.ts:7]: Now restricted it to `fields: ["name", "username"]`

**Pull request merged**